### PR TITLE
[macos] Mmp should quote paths when passing arguments to clang. Fixes… … #56850

### PR DESCRIPTION
**Pull request by timrisi**

### CODE REVIEW

File: tools/mmp/mmp.csproj [hunk 1]
```diff
@@ -334,6 +334,12 @@
     <Compile Include="..\common\Symbols.cs">
       <Link>external\Symbols.cs</Link>
     </Compile>
+    <Compile Include="..\common\StringUtils.cs">
+      <Link>external\StringUtils.cs</Link>
+    </Compile>
+    <Compile Include="..\common\StringUtils.cs">
```
Comment (rolfbjarne) [inline tools/mmp/mmp.csproj]: The file doesn't need to be included twice :smile:

**Comment by chamons** [comment]
My original call was "if you think it is totally safe and we get QA to validating the card before end of day Friday we can PR it".

There was an open question on if QA would get to said card, and if not it'd be pocket vetoed, but if you think it's too big Rolf I'm happy to go with your opinion.

**Comment by chamons** [comment]
Let's lean towards being risk adverse.

**Comment by spouliot** [comment]
Let's tag the bug fir 15.4

**Comment by chamons** [comment]
Done.

**Pull request closed without merging**